odoh.go

```diff
@@ -247,6 +259,68 @@ func UnmarshalObliviousDoHConfig(buffer []byte) (ObliviousDoHConfig, error) {
 	}, nil
 }
 
+type ObliviousDoHConfigs struct {
+	Configs []ObliviousDoHConfig
+}
+
+func CreateObliviousDoHConfigs(configs []ObliviousDoHConfig) ObliviousDoHConfigs {
+	return ObliviousDoHConfigs{
+		Configs: configs,
+	}
+}
+
+func (c ObliviousDoHConfigs) Marshal() []byte {
+	serializedConfigs := make([]byte, 0)
+	for _, config := range c.Configs {
+		serializedConfigs = append(serializedConfigs, config.Marshal()...)
+	}
+
+	buffer := make([]byte, 2)
+	binary.BigEndian.PutUint16(buffer[0:], uint16(len(serializedConfigs)))
```

Comment:
I'm not sure if the convention would be to put the number of configs, or the length of all the configs serialized, so for instance, if I had two configs of size 44 bytes each, then the number I would put her would be 2, instead of 88. Whether we put 2 or 88, deserialization shouldn't be a problem.

Reply:
Looking at `ObliviousDoHConfig ObliviousDoHConfigs<1..2^16-1>;` from the draft suggests that putting a number is more reasonable, otherwise this should've been specified as 
```
struct ObliviousDoHConfigs {
      opaque oblivious_doh_configs<1..2^16-1>,
}
```

Reply:
It's the length of all configs, not the number of configs. (That's just standard TLS notation.)